<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neuronvisio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Model</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Vector Control</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Variable to Record</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="var"/>
           </item>
           <item>
            <widget class="QPushButton" name="create_vector">
             <property name="text">
              <string>Create Vector</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="all_sections">
               <property name="text">
                <string>All Sections</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="selected_section">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Selected section</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="styleSheet">
             <string>border-color: rgb(0, 0, 0);</string>
            </property>
            <property name="title">
             <string>Run Control</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="init_btn">
               <property name="text">
                <string>Init </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QPushButton" name="run_btn">
               <property name="text">
                <string>Init + Run</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Initial voltage [mV]</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDoubleSpinBox" name="vSpinBox">
               <property name="toolTip">
                <string>Initial Voltage [mV]</string>
               </property>
               <property name="minimum">
                <double>-300.000000000000000</double>
               </property>
               <property name="maximum">
                <double>400.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>tStop [ms]</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QDoubleSpinBox" name="tstopSpinBox">
               <property name="suffix">
                <string/>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>dt [ms]</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QDoubleSpinBox" name="dtSpinBox">
               <property name="decimals">
                <number>4</number>
               </property>
               <property name="singleStep">
                <double>0.000100000000000</double>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Time [ms]</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QLabel" name="time_label">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>0.0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>3D control</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0" colspan="2">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>6</number>
               </property>
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <property name="fieldGrowthPolicy">
                  <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="variableToShowLabel">
                   <property name="text">
                    <string>Variable to show</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="varToShow">
                   <property name="text">
                    <string>v</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Start Value</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="startValue">
                   <property name="text">
                    <string>-90</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Ending Value</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="endValue">
                   <property name="text">
                    <string>40</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QSlider" name="timelineSlider">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="animationTime">
                   <property name="text">
                    <string>0.0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="Plot3D">
               <property name="text">
                <string>Plot3D</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Plots</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="plot_vector_btn">
            <property name="text">
             <string>plot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="fig_num_spinBox">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="legend">
            <property name="text">
             <string>legend</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="points">
            <property name="text">
             <string>points</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>298</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <column>
           <property name="text">
            <string>Vector</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Details</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Sec Info</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="sec_info_label">
          <property name="text">
           <string>No section selected.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <property name="toolTip">
        <string>ModelDb explorer</string>
       </property>
       <attribute name="title">
        <string>ModelDB Explorer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="filter_input"/>
          </item>
          <item>
           <widget class="QPushButton" name="filter_list_btn">
            <property name="toolTip">
             <string>Filter the models list using the given text.</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeWidget" name="tree_models">
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="animated">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="columnCount">
            <number>4</number>
           </property>
           <column>
            <property name="text">
             <string>Year</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Authors</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Title</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Id</string>
            </property>
           </column>
          </widget>
          <widget class="QTabWidget" name="tabWidget_2">
           <property name="tabPosition">
            <enum>QTabWidget::South</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_5">
            <attribute name="title">
             <string>Readme</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QTextBrowser" name="textBrowser_readme">
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_6">
            <attribute name="title">
             <string>Model Overview</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QTextBrowser" name="textBrowser_model_overview">
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>138</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="load_model_btn">
            <property name="toolTip">
             <string>Download, extract, compile and load the selected model.</string>
            </property>
            <property name="text">
             <string>Load Model</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../icons/res.qrc">
     <normaloff>:/icons/system-log-out.svg</normaloff>:/icons/system-log-out.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../icons/res.qrc">
     <normaloff>:/icons/help-browser.svg</normaloff>:/icons/help-browser.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../icons/res.qrc">
     <normaloff>:/icons/document-save.svg</normaloff>:/icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../icons/res.qrc">
     <normaloff>:/icons/document-open.svg</normaloff>:/icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
